src/metalperformanceshadersgraph.cs

```diff
@@ -0,0 +1,1840 @@
+
+		[Export ("runAsyncWithFeeds:targetTensors:targetOperations:executionDescriptor:")]
+		MPSGraphTensorDataDictionary RunAsync (MPSGraphTensorDataDictionary feeds, MPSGraphTensor[] targetTensors, [NullAllowed] MPSGraphOperation[] targetOperations, [NullAllowed] MPSGraphExecutionDescriptor executionDescriptor);
+
+		[Export ("runAsyncWithMTLCommandQueue:feeds:targetTensors:targetOperations:executionDescriptor:")]
+		MPSGraphTensorDataDictionary RunAsync (IMTLCommandQueue commandQueue, MPSGraphTensorDataDictionary feeds, MPSGraphTensor[] targetTensors, [NullAllowed] MPSGraphOperation[] targetOperations, [NullAllowed] MPSGraphExecutionDescriptor executionDescriptor);
+
+		[Export ("runAsyncWithMTLCommandQueue:feeds:targetOperations:resultsDictionary:executionDescriptor:")]
+		void RunAsync (IMTLCommandQueue commandQueue, MPSGraphTensorDataDictionary feeds, [NullAllowed] MPSGraphOperation[] targetOperations, MPSGraphTensorDataDictionary resultsDictionary, [NullAllowed] MPSGraphExecutionDescriptor executionDescriptor);
+
+		[Export ("encodeToCommandBuffer:feeds:targetTensors:targetOperations:executionDescriptor:")]
+		MPSGraphTensorDataDictionary Encode (MPSCommandBuffer commandBuffer, MPSGraphTensorDataDictionary feeds, MPSGraphTensor[] targetTensors, [NullAllowed] MPSGraphOperation[] targetOperations, [NullAllowed] MPSGraphExecutionDescriptor executionDescriptor);
+
+		[Export ("encodeToCommandBuffer:feeds:targetOperations:resultsDictionary:executionDescriptor:")]
+		void Encode (MPSCommandBuffer commandBuffer, MPSGraphTensorDataDictionary feeds, [NullAllowed] MPSGraphOperation[] targetOperations, MPSGraphTensorDataDictionary resultsDictionary, [NullAllowed] MPSGraphExecutionDescriptor executionDescriptor);
+
+	}
+
+	// @interface MPSGraphGradientOps (MPSGraph)
+	[iOS (14,0), TV (14,0), Mac (11,0), MacCatalyst (14,0)]
+	[Category]
+	[BaseType (typeof(MPSGraph))]
+	interface MPSGraph_MPSGraphGradientOps
+	{
+		// -(NSDictionary<MPSGraphTensor *,MPSGraphTensor *> * _Nonnull)gradientForPrimaryTensor:(MPSGraphTensor * _Nonnull)primaryTensor withTensors:(NSArray<MPSGraphTensor *> * _Nonnull)tensors name:(NSString * _Nullable)name __attribute__((swift_name("gradients(of:with:name:)")));
+		[Export ("gradientForPrimaryTensor:withTensors:name:")]
+		NSDictionary<MPSGraphTensor, MPSGraphTensor> Gradients (MPSGraphTensor of, MPSGraphTensor[] with, [NullAllowed] string name);
+	}
+
+	// @interface MPSGraphActivationOps (MPSGraph)
+	[iOS (14,0), TV (14,0), Mac (11,0), MacCatalyst (14,0)]
+	[Category]
+	[BaseType (typeof(MPSGraph))]
+	interface MPSGraph_MPSGraphActivationOps
+	{
+		// -(MPSGraphTensor * _Nonnull)reLUWithTensor:(MPSGraphTensor * _Nonnull)tensor name:(NSString * _Nullable)name;
+		[Export ("reLUWithTensor:name:")]
+		MPSGraphTensor ReLU (MPSGraphTensor tensor, [NullAllowed] string name);
```

Comment:
Would Get as a prefix make sense here also on the ones below?

```suggestion
		MPSGraphTensor GetReLU (MPSGraphTensor tensor, [NullAllowed] string name);
```

Reply:
No. The ops are nouns. This is the way the API was designed.

Reply:
One more argument (I don't know why I'm defending these names so strongly :-)) is that these are like the functions on the Math class. Like `Math.Sin`, `Math.Tan`, they don't use names like `GetSin` and `GetTan`. You can see similar patterns in System.Numerics.Tensors, the QNNX runtime, DiffSharp, etc.